demos/python_demos/common/models/__init__.py

```diff
@@ -29,4 +30,5 @@
     'RetinaFace',
     'SSD',
     'YOLO',
+    'SegmentationModel'
```

Comment:
```suggestion
    'SegmentationModel',
```

and also please keep the alphabetical order.